packages/dev/core/src/Materials/greasedLinePluginMaterial.ts

```diff
@@ -3,28 +3,41 @@ import { RawTexture } from "./Textures/rawTexture";
 import { MaterialPluginBase } from "./materialPluginBase";
 import type { Scene } from "../scene";
 import type { UniformBuffer } from "./uniformBuffer";
-import type { Vector2 } from "../Maths/math.vector";
-import { TmpVectors } from "../Maths/math.vector";
+import { Vector2, TmpVectors } from "../Maths/math.vector";
 import { Color3 } from "../Maths/math.color";
 import type { Nullable } from "../types";
 import type { Material } from "./material";
 import { MaterialDefines } from "./materialDefines";
 import type { AbstractMesh } from "../Meshes/abstractMesh";
 import type { BaseTexture } from "./Textures/baseTexture";
-import { DeepCopier } from "../Misc/deepCopier";
+// import { DeepCopier } from "../Misc/deepCopier";
 import { RegisterClass } from "../Misc/typeStore";
+import { DeepCopier } from "core/Misc/deepCopier";
 
+/**
+ * Material types for GreasedLine
+ * {@link https://doc.babylonjs.com/features/featuresDeepDive/mesh/creation/param/greased_line#materialtype}
+ */

```

Comment:
You also have to add a comment for each option!

Reply:
Commented.